From 008ff9c38ad91768bc5fee93b236795483a1112f Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Sun, 10 Jun 2018 17:34:04 +0800
Subject: [PATCH 22/23] libcutils: ignore a trivial error

Sometimes it complains: "add_tid_to_cgroup failed to write '1900'
(No space left on device); fd=11". I don't understand why.
The device should have enough space.

Just ignore this error.
---
 libcutils/sched_policy.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libcutils/sched_policy.cpp b/libcutils/sched_policy.cpp
index f72ec52d60..86872cb2a2 100644
--- a/libcutils/sched_policy.cpp
+++ b/libcutils/sched_policy.cpp
@@ -93,6 +93,8 @@ static int add_tid_to_cgroup(int tid, int fd)
                 return 0;
         SLOGW("add_tid_to_cgroup failed to write '%s' (%s); fd=%d\n",
               ptr, strerror(errno), fd);
+        if (errno == ENOSPC)
+                return 0;
         errno = EINVAL;
         return -1;
     }
-- 
2.17.1

